cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/model/staging/CVRClinicalRecord.java

```diff
@@ -370,11 +370,22 @@ public void setMSK_SLIDE_ID(String mskSlideID) {
         this.mskSlideID = mskSlideID;
     }
 
-    private String resolveSampleType(Integer isMetastasis) {
-        if (isMetastasis != null)
-            return isMetastasis == 0 ? "Primary" : "Metastasis";
+    public String resolveSampleType(Integer isMetastasis) {
```

Comment:
Sorry .. one more request ... the parameter to this function should probably be renamed to something like sampleType

Reply:
note: we decided to keep the field as is to match the CVR JSON field that we get the value from ("is_metastasis")

Reply:
after discussing, I understand that the DMP model still calls this field "isMetastasis", so the nomenclature carries through to this function call. Unless the DMP metadata model is updated (which maybe should be done) we can wait on making this change.